common/character_interactions/00_alliance.txt

```diff
@@ -106,8 +106,10 @@
 				text = can_call_in_civil_war_trigger_loc
 				subject = this
 				any_character_war = {
-					limit = { is_civil_war = yes }
-					can_call_in_civil_war_trigger = { CALLER = scope:actor ALLY = scope:recipient }
+					trigger_if = {
+						limit = { is_civil_war = yes }
+						can_call_in_civil_war_trigger = { CALLER = scope:actor ALLY = scope:recipient }
+					}
```

Comment:
Error log returns an error for `limit` in this file if it is not nested within `if` or `trigger_if`.

Reply:
Yeah, sorry. Didn't notice that it's a trigger (`any_`) at first. `limit` can't work here.

But the correct fix most likely will look like...
```suggestion
					OR = {
						is_civil_war = no
						can_call_in_civil_war_trigger = { CALLER = scope:actor ALLY = scope:recipient }
					}
```
It's risky to put `trigger_if` without `trigger_else` because it may create situation when there are no conditions.

In this example, if the war isn't civil (`is_civil_war = yes` returns false), there's empty condition.